plugins/@grouparoo/postgres/package.json

```diff
@@ -22,9 +22,9 @@
   },
   "scripts": {
     "prepare": "rm -rf dist && tsc --declaration",
-    "test": "jest",
+    "test": "jest ./__tests__",
```

Comment:
Isn't this the default?  I'd rather keep the `jest` command working out of the box.  If you really need to ignore a file, you can do that in `jest.config.js`

Reply:
Reverting